Microsoft.Toolkit.Uwp.UI/Cache/ImageCache.cs

```diff
@@ -34,11 +35,14 @@ public class ImageCache : CacheBase<BitmapImage>
         /// </summary>
         public static ImageCache Instance => _instance ?? (_instance = new ImageCache());
 
+        private DispatcherQueue _dispatcherQueue;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="ImageCache"/> class.
         /// </summary>
         public ImageCache()
```

Comment:
>ImageCache [](start = 15, length = 10)

Make sure to update docs and samples that this means ImageCache must not be created on a UI thread - that seems like it's likely a breaking change

Reply:
I've changed them so they can be set after creation as well. We'll call it out on the docs.